### PR TITLE
Update connector

### PR DESCRIPTION
**Pull request by RFlipper**
Update legal company name
Remove duplicated locales
Add new endpoints

### CODE REVIEW

File: certified-connectors/Plumsail Documents/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1280,9 +1294,7 @@
         "responses": {
           "200": {
             "description": "Success",
-            "schema": {
-              "type": "object"
-            }
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/Plumsail Documents/apiDefinition.swagger.json:1297]: @RFlipper empty schemas are not allowed. Please remove it or define the schema.

File: certified-connectors/Plumsail Documents/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1492,14 +1504,12 @@
...
-            "name": "request",
             "in": "body",
+            "name": "request",
             "required": true,
             "schema": {
               "type": "object",
-              "additionalProperties": {
-                "type": "object"
-              },
+              "additionalProperties": {},
```
Comment (v-EgorKozhadei) [inline certified-connectors/Plumsail Documents/apiDefinition.swagger.json:1512]: @RFlipper empty properties can be removed unless they're required.

File: certified-connectors/Plumsail Documents/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1,57 +1,37 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
     "title": "Plumsail Documents",
-    "description": "Generate documents from DOCX, XLSX, PPTX, HTML templates, and populate fillable PDF forms. Automate your contracts, invoices, reports, tickets, and more. Find more information at site https://plumsail.com/documents",
+    "description": "Plumsail Documents connector allows you to set up automated document generation and conversion. The connector also provides a wide range of actions to manipulate PDFs and PDF forms. Create documents from templates, convert to PDF,  split and merge PDF files, protect them, read PDF forms, and more - all you need to automate documents in one connector. Find more information at https://plumsail.com/documents.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Plumsail Documents/apiDefinition.swagger.json:5]: @RFlipper please make sure that everything is grammatically correct and is free of typographical errors.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @RFlipper,

Please check comments after the review.

Let us know if you have any questions.

**Comment by Amjed-Ayoub** [comment]
Hello @RFlipper,
I hope you are doing well.
I am following up with you to see how things are going with updating your connector.
Please let us know if you have any questions.
Thank you.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @RFlipper,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**